it/uk/gov/hmrc/agentsubscription/controllers/SubscriptionControllerForOverseasISpec.scala

```diff
@@ -162,6 +162,29 @@ class SubscriptionControllerForOverseasISpec extends BaseISpec with OverseasDesS
     }
 
     "return a 500 error if " when {
+      "the current application does not pass validation" in {
```

Comment:
looking at this test its not clear why exactly it is failing, perhaps add after not pass validation "because ..."

Reply:
Updated to be explicit